configure

```diff
@@ -542,7 +546,14 @@ def is_arm_hard_float_abi():
 def host_arch_cc():
   """Host architecture check using the CC command."""
 
-  k = cc_macros()
+  if sys.platform.startswith('aix'):
+    k = cc_macros('gcc')
+    matchup = {
+      '_ARCH_PPC'   : 'ppc',
+      '_ARCH_PPC64' : 'ppc64',
+    }
```

Comment:
I assume you just mean the matchup part. Good point. Since I thought we needed those lines I it seemed like I'd gotten it right when I moved them from the original list (they were removed from earlier patches because alphabetizing had caused issues)  .  I'll validate that I can just remove them and AIX still builds ok.